docs/UserGuide.adoc

```diff
@@ -95,10 +95,10 @@ Examples:
 
 * `edit 1 p/91234567 e/johndoe@example.com` +
 Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-* `edit 2 n/Betsy Crower t/` +
```

Comment:
It might be good to keep the original line because it it clearer (at least for me). So maybe we can do both? Something like:

`edit 2 n/Betsy Crower t\` **OR**
`e 2 n/Betsy Crower t\`

Just like how on lines 73 and 74, both examples are shown.

@CS2103JAN2018-F14-B2/developers Any other opinions on this?